cli/azd/internal/scaffold/scaffold_test.go

```diff
@@ -180,70 +184,6 @@ func TestExecInfra(t *testing.T) {
 				},
 			},
 		},
-		{
-			"API with MySQL",
```

Comment:
Kept one database test around, otherwise I think we're okay with the encompassing All at the moment.